flink-yarn/src/main/java/org/apache/flink/yarn/cli/FlinkYarnSessionCli.java

```diff
@@ -478,12 +485,15 @@ public static void runInteractiveCli(YarnClusterClient yarnCluster, boolean read
 
 	public static void main(final String[] args) throws Exception {
 		final FlinkYarnSessionCli cli = new FlinkYarnSessionCli("", ""); // no prefix for the YARN session
-		Configuration flinkConfiguration = GlobalConfiguration.loadConfiguration();
+
+		final String configurationDirectory = CliFrontend.getConfigurationDirectoryFromEnv();
```

Comment:
Maybe it should? It is kind of strange that part of the configuration is passed through `args` and some through `env`. I have never used this cli so maybe I have wrong feeling about it :)